articles/native-platforms/ios-objc/06-rules.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Rules
+description: This tutorial will show you how to create a basic rule that you can use in your app.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../includes/github', { link: 'https://github.com/auth0-samples/auth0-ios-objc-sample/tree/master/06-Rules', }) %>
```

Comment:
@pablolvillar @sebacancinos 
Both the directory and the filename are wrong and this causes the page to fail to load. The correct is `../../_includes/_github `